prow/github/client.go

```diff
@@ -2613,7 +2613,11 @@ func (c *client) ListTeamRepos(id int) ([]Repo, error) {
 			return &[]Repo{}
 		},
 		func(obj interface{}) {
-			repos = append(repos, *(obj.(*[]Repo))...)
+			for _, repo := range *obj.(*[]Repo) {
```

Comment:
This piece of logic needs a code comment explaining why we are doing this.

Reply:
Thank you for the review, and sorry I missed this bit that was pending me. I could not reopen the PR, so I will create a new one.